src/man/firejail-profile.txt

```diff
@@ -268,6 +268,8 @@ Use directory as user home.
 .TP
 \fBprivate-bin file,file
 Build a new /bin in a temporary filesystem, and copy the programs in the list.
+The files in the list must be expressed as relative to the /bin
```

Comment:
This could confuse user, because binaries under e.g. /usr/sbin are supported too.

Reply:
Ah, sorry -- I missed that.  I can update.  Is the full list:

* `/sbin
* `/usr/bin`
* `/usr/sbin`
* `/usr/local/bin`

?